src/components/Timer/Timer.tsx

```diff
@@ -86,6 +91,18 @@ export const Timer = (props: TimerProps) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [timeLeft]);
 
+  useEffect(() => {
+    if (isModerator && allReady && Object.values(timeLeft).some((time) => time > 0)) {
+      Toast.success(
+        <div>
+          <div>{i18n.t("Toast.allParticipantsDone")}</div>
```

Comment:
```suggestion
          <div>{t("Toast.allParticipantsDone")}</div>
```
Since we are doing `const {t} = useTranslation();` in line 29